### ISSUE TITLE
Upgrade fails - database migration InconsistentMigrationHistory

### ISSUE THREAD

**Issue by xf-** [getsentry/onpremise#1041]
### Version

21.8.0.dev 02a583e8

### Steps to Reproduce

1. Update via install.sh
2. Process throws an SIGERR

### Expected Result

Wokring upgrade & instance

### Upgrade Error
```log
11:03:59 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured.
11:04:01 [INFO] sentry.plugins.github: apps-not-configured
Running migrations for default
Traceback (most recent call last):
  File "/usr/local/bin/sentry", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/sentry/runner/__init__.py", line 196, in main
    func(**kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/sentry/runner/decorators.py", line 29, in inner
    return ctx.invoke(f, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/sentry/runner/commands/upgrade.py", line 100, in upgrade
    _upgrade(not noinput, traceback, verbosity, not no_repair, with_nodestore)
  File "/usr/local/lib/python3.6/site-packages/sentry/runner/commands/upgrade.py", line 56, in _upgrade
    verbosity=verbosity,
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 141, in call_command
    return command.execute(*args, **defaults)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 335, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 82, in handle
    executor.loader.check_consistent_history(connection)
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/loader.py", line 292, in check_consistent_history
    connection.alias,
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration jira_ac.0001_initial is applied before its dependency sentry.0001_squashed_0200_release_indices on database 'default'.
An error occurred, caught SIGERR on line 12
Cleaning up...
````

### Issue is persistent
Starting the instance and upgrading on another day will throw the same exception.

### Logging Fails
No new sentry entries, some views throwing errors.

## Similar issue on GitHub
- One has a fix
- Another was closed by author

**Comment by xf-** [comment]
Tried to destroy the setup and restore a backup. (`docker-compose down --rmi all -v --remove-orphans`)
Starting from 0 with Tag 21.7.0 or master will cause different migration errors.

django.db.migrations.exceptions.InconsistentMigrationHistory
```
Migration sentry.0201_semver_package is applied before its dependency sentry.0001_squashed_0200_release_indices
```

```
Migration  social_auth.0001_initial is applied before its dependency sentry.0001_squashed_0200_release_indices 
```

```
Migration admin.0001_initial is applied before its dependency sentry.0001_squashed_0200_release_indices 
```

**Comment by xpicio** [comment]
I'm upgrading from 21.6.1 to 21.7.0 and i'm facing the same issue but with the following exception

`django.db.migrations.exceptions.InconsistentMigrationHistory: Migration admin.0001_initial is applied before its dependency sentry.0001_squashed_0200_release_indices on database 'default'.`

**Comment by BYK** [comment]
Can you try going to `21.6.3` first and then going to `21.7.0`?

**Comment by xf-** [comment]
I'm back on 2.6.3 and i just started the update

**Comment by xf-** [comment]
Looks like it is working. i tried a lot of switching between master and 21.7.0, but 21.6.3 was last solution and I didn't wanted to touch it again.

**Comment by xpicio** [comment]
> Can you try going to `21.6.3` first and then going to `21.7.0`?

Upgrade from 21.6.1 to 21.6.3 and then to 21.7.0 is worked for me

**Comment by BYK** [comment]
@xf- have you tried the steps in https://develop.sentry.dev/self-hosted/troubleshooting/#kafka

Closing this issue as it is resolved with 21.6.3 -> 21.7.0 path.

**Comment by xf-** [comment]
I did that form the GitHub ticket. But tried the nuke option. Looks like a fix.
